skycoin/skywire
# Improve UI and Backend on `reboot` and `turn off`

Did you run `make format && make check`? Yes

Fixes #1667 

 Changes:	
- add `shutdown` endpoint to hypervisor API
- add `shutdown` bottom to UI (move all changes by senyoret from #1642)
- remove `reboot` from UI
- remove `restart` from skywire-cli and whole of system

How to test this PR:
- build ui by `make build-ui`
- build by `make build`
- create hypervisor config and run it
- check ui for removing `reboot` bottom and added `turn off` bottom there.

Note:
- based on #1667 we decided to remove reboot bottom from UI, because boards use `systemd` and turning visor off work like reboot, also for desktop visors we don't need reboot at all
- #1642 could close and just merge this (current) PR
